src/main/target/common_pre.h

```diff
@@ -382,9 +380,6 @@
 #define USE_RX_LINK_QUALITY_INFO
 #define USE_ESC_SENSOR_TELEMETRY
 #define USE_OSD_PROFILES
-#define USE_OSD_STICK_OVERLAY
```

Comment:
I'm also not sure about this part. I can still see some users use it for education purposes on some f411 AIO style quads. Just  remove it will make them stay on a old version of betafligjt permanently.